pipelines/pgbench/.ci/main.workflow

```diff
@@ -0,0 +1,15 @@
+workflow "Run Benchmarks" {
+  resolves = "analyse"
+}
+action "benchmark" {
+  uses = "actions/docker/cli@master"
+  runs = ["sh", "-c", "./pipelines/pgbench/benchmark.sh"]
+  env = {
+    PG_VERSIONS = "9.6,9.3"
```

Comment:
to make this even more generic, we can have a `PG_IMAGES` list, that would allow us to specify any docker image that contains postgres.